x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/action_delete.tsx

```diff
@@ -89,11 +100,19 @@ export const DeleteAction: FC<DeleteActionProps> = ({ items, forceDisable }) =>
   const bulkDeleteModalContent = (
     <>
       <p>
-        <FormattedMessage
-          id="xpack.transform.transformList.bulkDeleteModalBody"
-          defaultMessage="Are you sure you want to delete {count, plural, one {this} other {these}} {count} {count, plural, one {transform} other {transforms}}?"
-          values={{ count: items.length }}
-        />
+        {shouldForceDelete ? (
+          <FormattedMessage
+            id="xpack.transform.transformList.bulkForceDeleteModalBody"
+            defaultMessage="Are you sure you want to force delete {count, plural, one {this} other {these}} {count} {count, plural, one {transform} other {transforms}}? The {count, plural, one {transform} other {transforms}} will be deleted regardless of its current state."
```

Comment:
Ideally this should be `their current state` if more than one transform, and `its current state` if one transform.

Reply:
Good catch! Thanks Pete. Fixed in https://github.com/elastic/kibana/pull/69472/commits/33c2d9ab29b866853249776794048d984814d1bc